homeassistant/components/zwave.py

```diff
@@ -92,6 +93,10 @@
      [COMMAND_CLASS_DOOR_LOCK],
      TYPE_BOOL,
      GENRE_USER),
+    ('rollershutter',
+     [COMMAND_CLASS_SWITCH_MULTILEVEL],
```

Comment:
Are there any other ways to channel this correctly. I see there will be problems with identifying devices correctly to the correct component. I understood the as this:
If command class is switch multilevel AND contains a value with button AND has genre with user, it will be detected as rollershutter. 
Or have I misunderstood this? Open for suggestions :)